opae-libs/CMakeLists.txt

```diff
@@ -115,6 +115,9 @@ mark_as_advanced(OPAE_PRESERVE_REPOS)
 option(OPAE_BUILD_LIBOFS "Enable building of the libofs library and framework" ON)
 mark_as_advanced(OPAE_BUILD_LIBOFS)
 
+option(OPAE_BUILD_AFU_TEST "Enable building of afu_test" ON)
```

Comment:
Hi Tom, since this file lives in opae-libs, there needs to be a separate pull request filed at github.com/OPAE/opae-libs.git.